GoogleMapsComponents/Maps/BaseListableEntity.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.JSInterop;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GoogleMapsComponents.Maps
+{
+    public abstract class BaseListableEntity<T> : IDisposable, IJsObjectRef
```

Comment:
Maybe it is not big thing maybe yes. When i am not sure how to name class or anything i always looks at ms guys. There works pros. You know they call it ControllerBase, not BaseController. I think they spend days and many researches on such thing so you dont need to.

Reply:
True, absolutely true!
I'll rename it soon!
Put "Base" as suffix allow to keep file each near the other, too (alphabetical order)!